WordPress/Classes/ViewRelated/Domains/Domain registration/DomainPurchasingWebFlowController.swift

```diff
@@ -297,19 +348,21 @@ extension DomainPurchasingWebFlowController.DomainPurchasingError: CustomNSError
         case .invalidInput: return 500
         case .internal: return 501
         case .unsupportedRedirect: return 502
-        case .other(let error): return (error as NSError).code
+        case .other(let error): return 1000 + (error as NSError).code
```

Comment:
Not suggesting a change, but wondering (and might be missing something obvious), what is the significance of the `1000` here?

Reply:
Also, the whole `DomainPurchasingWebFlowController` is new and used only for `Site Creation > Domain Purchasing` feature. So refactoring anything in this class doesn't impact any existing feature.